java/ql/test/library-tests/dataflow/taint-format/test.ql

```diff
@@ -1,16 +1,18 @@
 import java
+import semmle.code.java.dataflow.DataFlow
 import semmle.code.java.dataflow.TaintTracking
+import TestUtilities.InlineFlowTest
 
-class Conf extends TaintTracking::Configuration {
-  Conf() { this = "qltest:dataflow:format" }
+class TaintFlowConf extends TaintTracking::Configuration {
+  TaintFlowConf() { this = "qltest:dataflow:format" }
 
   override predicate isSource(DataFlow::Node n) {
     n.asExpr().(MethodAccess).getMethod().hasName("taint")
   }
 
-  override predicate isSink(DataFlow::Node n) { any() }
+  override predicate isSink(DataFlow::Node n) { n instanceof DataFlow::ExprNode }
```

Comment:
Why are the sinks being restricted to `ExprNode` here?

Reply:
This was to account for the fact that `any()` includes hidden nodes that show up once you start modeling via the CSV framework. This code is now gone anyway due to some refactorings